nixos/modules/config/no-x-libs.nix

```diff
@@ -46,8 +46,9 @@ with lib;
       graphviz = super.graphviz-nox;
       gst_all_1 = super.gst_all_1 // {
         gst-plugins-bad = super.gst_all_1.gst-plugins-bad.override { guiSupport = false; };
-        gst-plugins-base = super.gst_all_1.gst-plugins-base.override { enableWayland = false; enableX11 = false; };
+        gst-plugins-base = super.gst_all_1.gst-plugins-base.override { enableGl = false; enableWayland = false; enableX11 = false; };
         gst-plugins-good = super.gst_all_1.gst-plugins-good.override { enableWayland = false; enableX11 = false; gtkSupport = false; qt5Support = false; qt6Support = false; };
+        gst-plugins-rs = super.gst_all_1.gst-plugins-rs.override { withGtkPlugins = false; };
```

Comment:
Nitpick, but `gst-plugins-good` has a "feature" called `gtkSupport`, but the new feature for `gst-plugin-rs` calls it `withGtkPlugins`.

Reply:
That's because it disabled 4 plugins. gtkSupport would more likely translate to just the gtk plugin.